src/system/PermissionsModule/Listener/Core3UpgradeListener.php

```diff
@@ -34,8 +34,20 @@ public function upgrade(GenericEvent $event): void
         if (!version_compare($event->getArgument('currentVersion'), '3.0.0', '<')) {
             return;
         }
+        if ($this->columnExists('group_perms', 'realm')) {
+            $sql = 'ALTER TABLE `group_perms` DROP COLUMN `realm`;';
+            $this->conn->executeQuery($sql);
+        }
         if ($this->columnExists('group_perms', 'bond')) {
-            $sql = 'ALTER TABLE `group_perms` DROP COLUMN `bond`';
+            $sql = 'ALTER TABLE `group_perms` DROP COLUMN `bond`;';
+            $this->conn->executeQuery($sql);
+        }
+        if ($this->columnExists('group_perms', 'comment')) {
```

Comment:
this won't work like you intend. you are basically saying, 'if the column exists, then add it'

Reply:
oops, missing `!`

Reply:
fixed, thanks